### PR TITLE
[TASK] Overhaul category type

### PR DESCRIPTION
**Pull request by linawolf**
* Correct indentation,
* add names for confvals

releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/ColumnsConfig/Type/Category/Properties/ForeignTablePrefix.rst [hunk 1]
```diff
@@ -5,8 +5,8 @@
 foreign_table_prefix
 ====================
 
-..  confval:: foreign_table_prefix (type => category)
-
+..  confval:: foreign_table_prefix
+    :name: category-properties-foreign-table-prefix
```
Comment (brotkrueml) [inline Documentation/ColumnsConfig/Type/Category/Properties/ForeignTablePrefix.rst:9]:
```suggestion
    :name: category-fforeign-table-prefix
```

Above you use "category-foreign-table-item-group", here you add "properties". Please be consistent. I suggest to remove "properties"

**Review by brotkrueml — COMMENTED** [review]
Only main and 12.4, as category was introduced with 12.0

**Comment by linawolf** [comment]
According to the index page categories were introduced with 11.4

File: Documentation/ColumnsConfig/Type/Category/Properties/ForeignTablePrefix.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/ColumnsConfig/Type/Category/Properties/ForeignTablePrefix.rst]: Copy & Pasted that accidently

**Comment by linawolf** [comment]
Yep: https://docs.typo3.org/m/typo3/reference-tca/11.5/en-us/ColumnsConfig/Type/Category/Index.html

**Comment by brotkrueml** [comment]
> Yep: https://docs.typo3.org/m/typo3/reference-tca/11.5/en-us/ColumnsConfig/Type/Category/Index.html

Ups, you are right. Sorry for the noise.

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**